docs/user_documentation/guide-questionnaire.md

```diff
@@ -37,4 +37,28 @@ Go to the data explorer and look for the (in our case) Simple Questionnaire tabl
 
 This table contains the answers for all the users that filled in the questionnaire.
 
+# Configure PDF report
+After submitting a questionnaire users can download a PDF report of their data. All questions and answers that were provided will be presented nicely formatted in this report.  
+This PDF report has an optional intro text and logo. To provide these, an extra attribute, with the name "report_header" should be added to the questionnaire. This attribute 
+should be of datatype "xref" with as refEntity a table that specifies the "intro" and "logo". To have the same logo and intro for each participant, use the defaultValue option to
+specify the data row in the config table. Example structure:
  
+Attributes:
+ 
+| name               | entity               | dataType               | idAttribute | labelAttribute | defaultValue | refEntity          | label                                           | partOfAttribute |
+|--------------------|----------------------|------------------------|-------------|----------------|--------------|--------------------|-------------------------------------------------|-----------------|
+| id                 | demo_report_header   | string                 | TRUE        | TRUE           |              |                    |                                                 |                 |
```

Comment:
Did you consider making the questionnaire report a system entity type?:
- Easier for users to create reports
- Enforce cascading deletes for data cleanup
- Data managers are more likely to use feature if it is less hidden

Reply:
@ConnorStroomberg could you add your standup reply to my question?

Reply:
The single data manager (user) using this feature, suggested the proposed setup. The proposed setup works and is simper than adding a  system entity type. I suggest to use the current setup and change to using system entity if additional features are requested.